week-1/2-mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
-function getNumber() {
-  return Math.random() * 10;
+function letterSet() {
+  return Math.random() * 10; //Math.random() syntax is a floating-point, pseudo-random number between 0 (inclusive) and 1 (exclusive).
```

Comment:
What you've said here is correct but what else is happening here (look at the `* 10`)?

Reply:
Thank you @Pads. Each time the code is ran, it will return random values 0 to 9. Result: 0, 1, 2, 3, 4, 5, 6, 7, 8, 9.